File: HomeSmart/HomeSmart/Model/Garden.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HomeSmart.Model
{
    public class Garden
    {
        public string _id { get; set; }
        public string homeId { get; set; }
        public int temperature { get; set; }
        public int humidity { get; set; }
        public Boolean waterPump { get; set; }
        public DateTime createdAt { get; set; }
        public DateTime updatedAt { get; set; }
    }
}

File: Server/models/Garden.js
const mongoose = require('mongoose')

const GardenSchema = new mongoose.Schema({
    homeId:{
        type: String,
        required: true
    },
    temperature:{
        type: Number,
        default: 30,
    },
    humidity:{
        type: Number,
        default: 75,
    },
    waterPump:{
        type: Boolean,
        default: false,
        // false: Tắt, True: Bật
    }

})

module.exports = mongoose.model('Garden',GardenSchema)